.nix/unionfind.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchurl, buildDunePackage }:
+
+buildDunePackage rec {
```

Comment:
Same question than bindlib: why do you need this file + duplication of version numbers, etc.